compiler/rustc_codegen_cranelift/src/inline_asm.rs

```diff
@@ -462,8 +465,12 @@ impl<'tcx> InlineAssemblyGenerator<'_, 'tcx> {
         let mut slots_output = vec![None; self.operands.len()];
 
         let new_slot_fn = |slot_size: &mut Size, reg_class: InlineAsmRegClass| {
-            let reg_size =
-                reg_class.supported_types(self.arch).iter().map(|(ty, _)| ty.size()).max().unwrap();
+            let reg_size = reg_class
+                .supported_types(self.arch, self.allow_experimental_reg)
```

Comment:
At the codegen stage we can just use `true` here instead of `allowed_experimental_reg`. All the stability checking will have been done in priot stages.

Reply:
Changed rustc_codegen_* to just use true.

I also left a document about the cases in which it is fine to always pass true.

https://github.com/rust-lang/rust/blob/c024d8ccdfb462f4e99b54e6c8456fbc4d745752/compiler/rustc_target/src/asm/mod.rs#L608-L610